services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -203,18 +210,20 @@ async def get_for_user_or_workspace(
     else:
         query = query.where(folders_v2.c.workspace_id == workspace_id)
 
-    async with get_database_engine(app).acquire() as conn:
+    async with pass_or_acquire_connection(get_asyncpg_engine(app), connection) as conn:
         result = await conn.execute(query)
-        row = await result.first()
+        row = result.first()
```

Comment:
And again here. I would expect this one to be await maybe the `execute` above should be a `stream`?